packages/@glimmer/compiler/lib/PASSES.md

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
This document is great, thank you for putting it together! 

Maybe it should move to `packages/@glimmer/compiler/README.md`?

Reply:
Seems legit. My main concern was that there's more going on in the whole package than just the passes, and wanted to be clear about what each directory is doing, but I'd happily move it into a root README (or perhaps a link off the root README?)